### ISSUE TITLE
Add. 18,819, foll. 1–91 | https://bl.vuexistapps.us/manuscript/1128 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1078]
### Problems with Manuscript Overview

- Date: the Roman numerals appear in Upper Case, and the word "Century" is not abbreviated as usual. 
- Wright mentions two sets of dates for this MS. The majority are written in ixth or xth, while 15 folios were written in xiith or xiiith. Only the major date is reflected in the MS overview. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [X] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### General Notes

- Folio number missing: "Addition 1 (Fol. )"

**Comment by mikey-tang16** [comment]
@wlpotter Should we add the minor date 12th/13th century to the overview section? If so, how should I represent that?

**Comment by wlpotter** [comment]
No, we want only the earliest production date to be listed. It looks like in this case the 12th/13th c. hand are a later repair. We include that date in the hand note so users know about it, but we only record the original 9th/10th c. date as the `origDate`
